docs/core/extensions/snippets/logging/library-authors/LibraryConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Logging.LibraryAuthors;
+
+public sealed class LibraryConfiguration
```

Comment:
It wasn't clear to me how devs reading the docs would get to this file? I saw a reference to the LogMessages.cs file, but not to these others. I'm hoping we can make it clear that this API is an example, but we aren't being overly prescriptive about the shape of the API library authors use. I hope library authors choose something that fits the style of their library's pre-existing API which might be a static method, static property, a parameter on a constructor, parameters on multiple constructors, instance methods or properties on singletons or builder-style objects, etc.

Reply:
It's not exposed in the doc, it was just working code when trying to conceptualize the guidance. Do you think it's necessary to show this? If not, we can remove it.